catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -0,0 +1,302 @@
+
+        addMetacardSubject(pathValueTracker, metacard);
+
+        return metacard;
+    }
+
+    private void addMetacardDates(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+
+        String dateStr = pathValueTracker.getPathValue(DATE_STAMP_PATH);
+        if (StringUtils.isNotEmpty(dateStr)) {
+            Date date = CswUnmarshallHelper.convertToDate(dateStr);
+            metacard.setModifiedDate(date);
+            metacard.setCreatedDate(date);
+        }
+
+    }
+
+    private void addMetacardType(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+
+        String type = pathValueTracker.getPathValue(CODE_LIST_VALUE_PATH);
+        if (StringUtils.isNotEmpty(type)) {
+            metacard.setContentTypeName(type);
+            metacard.setAttribute(GmdMetacardType.GMD_TYPE, type);
+        }
+
+    }
+
+    private void addMetacardCrs(final XstreamPathValueTracker pathValueTracker, MetacardImpl metacard) {
+        StringBuilder crs = new StringBuilder();
+        crs.append("urn:ogc:def:crs:");
+        crs.append(StringUtils.defaultString(pathValueTracker.getPathValue(CRS_AUTHORITY_PATH)));
+        crs.append(COLON);
+        crs.append(StringUtils.defaultString(pathValueTracker.getPathValue(CRS_VERSION_PATH)));
+        crs.append(COLON);
+        crs.append(StringUtils.defaultString(pathValueTracker.getPathValue(CRS_CODE_PATH)));
+        metacard.setAttribute(GmdMetacardType.GMD_CRS, crs.toString());
+
+    }
+
+    private void addMetacardTitle(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        String title = pathValueTracker.getPathValue(TITLE_PATH);
+        if (StringUtils.isNotEmpty(title)) {
+            metacard.setTitle(title);
+        }
+    }
+
+    private void addMetacardAbstract(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        String gmdAbstract = pathValueTracker.getPathValue(ABSTRACT_PATH);
+        if (StringUtils.isNotEmpty(gmdAbstract)) {
+            metacard.setAttribute(GmdMetacardType.GMD_ABSTRACT, gmdAbstract);
+        }
+
+    }
+
+    private void addMetacardFormat(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        String format = pathValueTracker.getPathValue(FORMAT_PATH);
+        if (StringUtils.isNotEmpty((format))) {
+            metacard.setAttribute(GmdMetacardType.GMD_FORMAT, format);
+        }
+    }
+
+    private void addMetacardResourceUri(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        String linkage = pathValueTracker.getPathValue(LINKAGE_URI_PATH);
+        if (StringUtils.isNotEmpty(linkage)) {
+            try {
+                metacard.setResourceURI(new URI(linkage));
+            } catch (URISyntaxException e) {
+                LOGGER.info("Unable to read resource URI {]", linkage);
```

Comment:
Incorrect usage of parameter replacement. Should be `{}`. Ex: `logger.debug("Temperature set to {}. Old temperature was {}.", t, oldT);`